docs/nuget-org/policies/deleting-packages.md

```diff
@@ -34,3 +34,27 @@ Packages that meet any of the following criteria are not allowed on the public N
 If you find a package that is in violation of any of these items, click the **Report Abuse** link on the package details page and submit a report.
 
 Note that the NuGet team and the .NET Foundation reserves the right to change these criteria at any time.
+
+#### Unlisting a package
+In order to unlist a specific package version please follow these steps:
+
+** Create a API Key **
+- Login to your Nuget.org organization account (click the "Sign in" link at the top right corner at nuget.org.
+- You will now see your organization name instead of the "Sign in" link at the top right corner. 
+- Click the link that displays your organization name and chose "API Keys"
+- You will now see "Create" and "Manage" sections. Click the plus sign beside the "Create" section.
+- Provide a "Key name" and make sure to check the “Unlist package” checkbox. 
+- In the “Available packages” check which of your nuget package(s) you want to unlist. 
+- Click the “Create” button. 
+- Under the ”Manage” section you will now be able to copy your newly created "API Key". Make sure to save it, since you will not be go back and copy it. 
```

Comment:
Could you explain why we have the "Create a API key" section? You should be able to unlist a package all in the nuget.org UI without an API key as you mentioned in the next section.